rollup.config.js

```diff
@@ -65,7 +67,7 @@ export default args => {
                     preventAssignment: true
                 }),
                 eslint(),
-                typescript({ target })
+                typescript({ target, downlevelIteration })
```

Comment:
What was the issue?

Reply:
simply not working, it said me that iteration was not able in ES5 and proposed me this solution. I invite you to test by remove this line (we have certainly this issue since a log time, we test not enough this feature ;-) )